admin/ucf-degree-config.php

```diff
@@ -193,6 +263,7 @@ public static function display_settings_field( $args ) {
 			$option_name   = $args['label_for'];
 			$description   = $args['description'];
 			$field_type    = $args['type'];
+			$options       = $args['options'];
```

Comment:
`$args['options']` isn't set on basic text fields and will throw undefined index notices on fields that don't have it defined